src/css/styles.css

```diff
@@ -345,3 +319,10 @@ iframe {
 .login-form__footer {
     text-align: right;
 }
+
+.loading {
+  margin-left: 20px;
+  height: 70px;
+  background-image: url(../img/infinity-loader.svg);
+  background-repeat: no-repeat;
```

Comment:
Sure, I'm a terrible noob when talking CSS, could you please share a link about the differences?